src/protocols/LoRaWAN/LoRaWAN.cpp

```diff
@@ -713,18 +893,23 @@ int16_t LoRaWANNode::downlink(uint8_t* data, size_t* len) {
     RADIOLIB_ASSERT(state);
   }
 
-  // calculate the channel scanning timeout
-  // according to the spec, this must be at least enough time to effectively detect a preamble
-  uint32_t scanTimeout = this->phyLayer->getTimeOnAir(0)/1000;
+  // create the masks that are required for receiving downlinks
+  uint16_t irqFlags;
+  uint16_t irqMask;
```

Comment:
It's a good practice to always explicitly initialize variables changed by reference to a known safe value - at the moment there is no execution path that leaves them unitialized, but that may change in the future.